exp-player/addon/components/exp-lookit-geometry-alternation/component.js

```diff
@@ -550,7 +550,7 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
          *
          * @event startIntro
          */
-        _this.sendTimeEvent('startIntro');
+        _this.send('setTimeEvent', 'exp-alternation:startIntro');
```

Comment:
Whoops, that is not intentional - if you're up for changing this that'd be great. None of the analysis pipeline is set up yet anyway, so no problem making it robust to a few different names for the event.